crates/astria-conductor/src/conductor.rs

```diff
@@ -124,14 +143,33 @@ impl Conductor {
         // Spawn the sequencer task
         // Only spawn the sequencer::Reader if CommitLevel is not FirmOnly, also
         // send () to sync_done to start normal block execution behavior
-        let mut sync_done = futures::future::Fuse::terminated();
+        let mut seq_sync_done = futures::future::Fuse::terminated();
+        let mut da_sync_done = futures::future::Fuse::terminated();
 
-        // if only using firm blocks
-        if cfg.execution_commit_level.is_firm_only() {
-            // kill the sync to just run normally
-            let (sync_done_tx, sync_done_rx) = oneshot::channel();
-            sync_done = sync_done_rx.fuse();
-            let _ = sync_done_tx.send(());
+        let mut sequencer_reader = None;
+
+        info!("Conductor commit level: {:?}", cfg.execution_commit_level);
+
+        match cfg.execution_commit_level {
+            CommitLevel::SoftOnly => {
+                info!("only syncing from sequencer");
+                // kill the DA sync to only execute from sequencer
+                let (sync_done_tx, sync_done_rx) = oneshot::channel();
+                da_sync_done = sync_done_rx.fuse();
+                let _ = sync_done_tx.send(());
+            }
+            CommitLevel::SoftAndFirm => {
+                info!("syncing from DA then sequencer");
+                // when running in soft and firm mode, a sync cycle from both DA
+                // and sequencer are used. No need to kill any of the syncs
```

Comment:
why do we need to sync blocks from both? isn't it sufficient to sync from block 1 -> firm from DA, then firm -> soft from sequencer?

Reply:
Yeah that was my intention. I'll update the comment to reflect that.